independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json

```diff
@@ -0,0 +1,9216 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Influenza and Covid-19",
+    "description": "Influenza and Covid-19 Tracker provides real-time disease-related statistics accumulated worldwide",
+    "version": "1.0",
+    "contact": {
+      "name": "Kevin Comba Gatimu and Denis Wachira Kathuri",
+      "url": "https://linktr.ee/kevincomba",
+      "email": "kelchospenseworks247@gmail.com"
+    }
+  },
+  "host": "disease.sh",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/v3/covid-19/states": {
+      "get": {
+        "responses": {
+          "default": {
```

Comment:
@kelcho-spense Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**

Reply:
I have rectified the error.